src/main/fc/fc_rc.c

```diff
@@ -118,16 +118,24 @@ static void calculateSetpointRate(int axis)
     rcDeflection[axis] = rcCommandf;
     const float rcCommandfAbs = ABS(rcCommandf);
     rcDeflectionAbs[axis] = rcCommandfAbs;
+    float angleRate;
 
-    if (rcExpo) {
-        const float expof = rcExpo / 100.0f;
-        rcCommandf = rcCommandf * power3(rcCommandfAbs) * expof + rcCommandf * (1-expof);
-    }
+    if (currentControlRateProfile->rfRatesEnabled) {
+        // -1.0 to 1.0 ranged and curved 
+        rcCommandf = ((1.0f + 0.01f * (float)currentControlRateProfile->rfExpo[axis] * (rcCommandf * rcCommandf - 1.0f)) * rcCommandf);
```

Comment:
this line is interpolating between `rcCommandf` and `rcCommandf^3` (using `rfExpo/100`), but it's extremely hard to decipher .. 
```
a = currentControlRateProfile->rfExpo[axis] * 0.01;
rcCommandf = (1 - a) * rcCommandf + a * rcCommandf ^ 3;